x/gmm/client/cli/tx_create_pool.go

```diff
@@ -0,0 +1,126 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/sideprotocol/side/x/gmm/types"
+	"github.com/spf13/cobra"
+)
+
+var _ = strconv.Itoa(0)
+
+func CmdCreatePool() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create-pool [tokens] [decimals] [weights] [swap-fee] ",
+		Short: "Broadcast message create-pool",
+		Args:  cobra.ExactArgs(4),
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			tokens, err := sdk.ParseCoinsNormalized(args[0])
+			if err != nil {
+				return err
+			}
+
+			weights, err := parseWeights(args[2])
+			if err != nil {
+				return err
+			}
+
+			decimals, err := parseDecimals(args[3])
+			if err != nil {
+				return err
+			}
+
+			swapFee, err := strconv.Atoi(args[4])
+			if err != nil {
+				return err
+			}
+			if swapFee > 10000 {
+				return fmt.Errorf("swap fee must be less than 10000")
+			}
```

Comment:
'10000' is an amount or an percent ?

Reply:
percent. in ibcswap, we've used 0-10000 value. I guess it mentioned on ur spec.

Reply:
ok.  would be great adding comments.